### ISSUE TITLE
markbot issue

### ISSUE THREAD

**Issue by jsurtees** [jsurtees/content-semantics#1]
@thomasjbradley

I think I've figured it out. Markbot was suggesting 2x the proper indentation (it would suggest 8 but the proper number was 4 etc.). Thank you for the info on finding the issues tab.

**Comment by thomasjbradley** [comment]
Yeah, it looks like you figured it out. I just had a look at the code and it looks perfect.
